src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -299,18 +744,47 @@
 		"sourcePath" : "log.sagebase.org/aclSnapshots/",
 		"tableDescriptor": {
 			"name": "aclsnapshots",
-			"columns": {
-				"change_timestamp": "timestamp",
-				"change_type": "string",
-				"snapshot_timestamp": "timestamp",
-				"owner_id": "bigint",
-				"owner_type": "string",
-				"created_on": "timestamp",
-				"resource_access": "array<struct<principalId:bigint,accessType:array<string>>>"
-			},
-			"partitionKeys": {
-				"snapshot_date": "date"
+			"description" : "This table contain snapshots of access-control-list. Snapshots are taken when acl is created, updated or deleted. Note: Snapshots are also taken periodically and independently of the changes. The snapshot_timestamp records when the snapshot was taken.",
```

Comment:
Same comment as above about distinguishing between the periodic snapshots vs node updates.

Reply:
Same explanation as above